subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -81,6 +81,11 @@
 #define PDU_DC_PAYLOAD_TIME_MIN 328
 #define PDU_DC_PAYLOAD_TIME_MIN_CODED 2704
 
+/* Data channel maximum payload size and time */
+#define PDU_DC_PAYLOAD_SIZE_MAX 251
+#define PDU_DC_PAYLOAD_TIME_MAX 2120
```

Comment:
what about CTE and 2128us?

Reply:
In my humble opinion this value should be conditionally compiled. Is should depend on enabled features. If direction finding is enabled then it should be 2128 (BT Core 5.3 Vol 6, Part B section 4.5.10 table 4.6).

The additional 8us are related with CTEInfo that is part of a PDUs headed for LL Data PDU. If DF is enabled the header with CTEInfo has 24bits. That does not affect size of payload of a LL DATA PDU. (BT Core 5.3 Vol 6, Part B, section 2.4), hence the total max time for TX a PDU with PHY1M is 2128us.